crates/astria-sequencer/src/service/mempool.rs

```diff
@@ -82,6 +84,14 @@ async fn handle_check_tx<S: StateReadExt + 'static>(
     let request::CheckTx {
         tx, ..
     } = req;
+    if tx.len() > MAX_TX_SIZE {
+        return response::CheckTx {
+            code: AbciCode::INVALID_PARAMETER.into(),
```

Comment:
Can we add a new AbciCode for this? Will make it easier to understand what went wrong in logs without hving to parse the log